recipes/otsvm/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBUILD_DOC=OFF \
+  ..
+
+make install
+DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib ctest -R pyinstallcheck -E saveload --output-on-failure ${MAKEFLAGS}
```

Comment:
@patricksnape, I explicitely removed newlines because of previsous remarks from @jakirkham on my previous PRs, at some point we must converge on this, maybe a rule should be set in the linter ?